Assets/MixedRealityToolkit.Tests/PlayModeTests/PressableButtonTests.cs

```diff
@@ -28,11 +31,43 @@ public void Setup()
         }
 
         #region Utilities
-        private GameObject InstantiateDefaultPressableButton()
+
+        private static string PrefabDirectoryPath = "Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs";
+
+        private static Dictionary<string, bool> PressableButtonTestPrefabs = new Dictionary<string, bool>
+        {
+            // Key is file name.  Value is whether or not it needs to be placed in a Canvas.
+            { "PressableButtonHoloLens2.prefab", false },
+            { "PressableButtonHoloLens2UnityUI.prefab", true },
+        };
+
+        public static IEnumerable<string> PressableButtonsTestPrefabFilenames
         {
-            Object pressableButtonPrefab = AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/PressableButtonHoloLens2.prefab", typeof(Object));
+            get
+            {
+                foreach (var prefabFilename in PressableButtonTestPrefabs.Keys)
+                {
+                    yield return prefabFilename;
+                }
+            }
+        }
+
+        private GameObject InstantiateDefaultPressableButton(string prefabFilename)
+        {
+            var path = Path.Combine(PrefabDirectoryPath, prefabFilename);
+            Object pressableButtonPrefab = AssetDatabase.LoadAssetAtPath(path, typeof(Object));
             GameObject testButton = Object.Instantiate(pressableButtonPrefab) as GameObject;
 
+            if (PressableButtonTestPrefabs[prefabFilename])
+            {
+                //TODO: does this get cleaned up by TearDown?
```

Comment:
should be safe to not destroy this as the play test base class is deleting all the root level gameobjects in the tests scenes on teardown

Reply:
Thanks.  Yes I ran all of the tests with my latest changes.  They all pass and none leave any gameobjects behind.